builtins/en/facets/instructions/fix.md

```
@@ -30,6 +35,6 @@ Use reports in the Report Directory and fix the issues raised by the reviewer.
 |--------|-------|
 | new (fixed in this iteration) | {N} |
 | reopened (recurrence fixed) | {N} |
-| persists (carried over, not addressed this iteration) | {N} |
+| persists (carried over, fixed in this iteration) | {N} |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)